calendar.html

```diff
@@ -0,0 +1,64 @@
+---
+layout: plain
+title: An Open Source Community
```

Comment:
Change it to `Community Calendar`

Reply:
Are you referring to renaming the button to Community Calendar?

Reply:
The title. 
From `An open source community` to `Commmunity Calendar`.

Reply:
Oh, alright, I'll change it.